app/assets/stylesheets/application.css

```diff
@@ -1,15 +1,290 @@
+.create-category h1 {
+  margin: auto;
+}
+
+.create-transaction nav {
+  justify-content: space-between;
+}
+
+.welcome-wrapper {
+  padding: 3rem;
+  background: var(--main-color);
+}
+
+.welcome-page {
+  background: var(--general);
+  color: black;
+  justify-content: space-around;
+  align-items: center;
+  border-radius: 1rem;
+}
+
+.welcome-page img {
+  width: 5rem;
+  animation: logo 0.9s ease-in-out infinite both;
+}
+
+.welcome-btn-container {
+  gap: 1rem;
+}
+
+.welcome-msg {
+  text-align: center;
+  width: 70%;
+}
+
+.app-name {
+  font-weight: 900;
+  font-size: 2.5rem;
+}
+
+.item-card {
+  background: var(--general);
+  margin: 1rem 0;
+  justify-content: space-between;
+  border-radius: 5px;
+  align-items: center;
+  padding: 0.5rem;
+}
+
+.categories-page {
+  display: flex;
+  align-items: center;
+}
+
+.create-container {
+  width: 100%;
+  position: fixed;
+  bottom: 0;
+  padding: 1rem;
+}
+
+.temp-msg {
+  text-align: center;
+}
+
+.create-container button,
+.welcome-btn-container button {
+  background: var(--secondary);
+  border: none;
+  color: var(--general);
+  border-radius: 5px;
+  max-width: 100%;
+  height: 3.5rem;
+  padding: 0.5rem;
+}
+
+.welcome-btn-container a {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.welcome-btn-container button {
+  width: 60%;
+}
+
+.form-container button,
+input[type="submit"] {
+  border: none;
+  background-color: var(--secondary);
+  padding: 8px;
+  color: var(--header-color);
+}
+
+#checkbox {
+  justify-content: center;
+  align-items: center;
+  gap: 0.5rem;
+  border-bottom: none;
+}
+
+#checkbox input {
+  width: 1rem;
+}
+
+form :nth-last-child(n + 2) {
+  height: 3rem;
+  border: none;
+  border-bottom: 0.5px solid rgba(0, 0, 0, 0.25);
+  width: 100%;
+  padding: 0.5rem;
+}
+
+form :nth-last-child(n + 2):focus {
+  outline: none;
+}
+
+.reset form,
+.add-new form {
+  width: 100%;
+}
+
+.reset h2 {
+  margin: 1rem 0;
+}
+
+.links {
+  gap: 1rem;
+}
+
+.reset .links {
+  gap: 0;
+}
+
+.actions {
+  justify-content: center;
+}
+
+.reset .actions {
+  align-items: center;
+}
+
+.categories-container {
+  margin: 0.5rem 0;
+  max-height: 30rem;
+  padding: 0 2rem;
+  overflow-y: scroll;
+}
+
+.total-amount {
+  padding: 1rem;
+  background: var(--general);
+  justify-content: space-between;
+  box-shadow: 1px 1px 3px var(--text-color);
+}
+
+.form-container {
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  padding-bottom: 5rem;
+}
+
+.form-container input[type="submit"],
+.form-container button {
+  color: var(--general);
+  width: 60%;
+  margin: 1rem 0 0;
+  border-radius: 0;
+  height: 3rem;
+}
+
+.links a {
+  display: flex;
+  justify-content: center;
+  text-decoration: none;
+}
+
+/* @-webkit-keyframes logo{
+  0% {
+    -webkit-transform:scale(1);transform:scale(1)
+  }
+  50% {
+    -webkit-transform:scale(1.1);transform:scale(1.1)
+  }
+  100% {-webkit-transform:scale(1);transform:scale(1)
+  }
+} */
```

Comment:
* Kindly remove all commented code from your codebase as it makes your code look bulky 🙏